src/credix_lp/depository.ts

```diff
@@ -0,0 +1,286 @@
+import { BorshAccountsCoder, Wallet } from '@project-serum/anchor';
+import { ConfirmOptions, Connection, PublicKey, Signer } from '@solana/web3.js';
+import { IDL } from '../idl';
+import { CredixLpDepositoryAccount } from '../interfaces';
+import { Token, TOKEN_PROGRAM_ID } from '@solana/spl-token';
+import { findATAAddrSync } from '../utils';
+import { AnchorProvider } from '@project-serum/anchor';
+import { IDL as credixIDL, Credix as CredixIDL } from './credixIdl';
+import { Program } from '@project-serum/anchor';
```

Comment:
nit, `'@project-serum/anchor'` is imported twice:

```suggestion
import { AnchorProvider, Program } from '@project-serum/anchor';
import { IDL as credixIDL, Credix as CredixIDL } from './credixIdl';
```

Reply:
👌